fulcrumapp/fulcrum-schema
# story/FLCRM-9629 Expose record links table

**Proposed Changes:**

Makes the "record_links" accessible via the Query API. The tables referenced in this PR were applied to the database in https://fulcrumapp.atlassian.net/browse/FLCRM-9606 and https://fulcrumapp.atlassian.net/browse/FLCRM-9784. 

**Testing Steps:**

To test this:

- Find the `version` in the `publish-npm` step of the Tekton pipeline run for this PR
- Set the version as the value of the `@fulcrumapp/fulcrum-schema` dependency in the package.json of the schema-service
- Skaffold the schema-service and wait for it to come up
- Assuming https://github.com/fulcrumapp/query/pull/196 has been deployed to Query, you should now be able to run these queries (via Postman or curl):
- `select * from tables` - you should see "record_links" in the results
- `select * from record_links` - you should at least see the fields returned, rather than an error

I performed this in my namespace (toddrun) instead of the preview env, but was able to query for link data:
![Screen Shot 2022-08-15 at 1 02 09 PM](https://user-images.githubusercontent.com/2982313/184708887-2ce0d869-83a2-4232-b50d-4802cd56182d.png)
